src/main/java/net/snowflake/ingest/streaming/internal/BlobLatencies.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright (c) 2023 Snowflake Computing Inc. All rights reserved.
+ */
+
+package net.snowflake.ingest.streaming.internal;
+
+import com.codahale.metrics.Timer;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.util.concurrent.TimeUnit;
+
+/** Latency information for a blob */
+class BlobLatencies {
+  public static final long DEFAULT_BLOB_LATENCY = -1;
+
+  private long buildLatencyMs;
+  private long uploadLatencyMs;
+
+  private long flushStartTimestamp;
+  private long registerStartTimestamp;
+
+  public BlobLatencies() {
+    this.buildLatencyMs = DEFAULT_BLOB_LATENCY;
+    this.uploadLatencyMs = DEFAULT_BLOB_LATENCY;
+
+    this.flushStartTimestamp = DEFAULT_BLOB_LATENCY;
+    this.registerStartTimestamp = DEFAULT_BLOB_LATENCY;
```

Comment:
Do these need to be initialized? Why not use the default value of `0`?

Reply:
They do not neeeed to be, but I want a value that indicates error. It will reduce confusion - so we know if we are missing the blob latency or if it was actually 0.